### PR TITLE
Add call to the backend checks API

### PR DESCRIPTION
**Pull request by MancunianSam**
This can't be deployed until it's time to switch everything over to the
new backend checks process otherwise we'll end up with duplicate
metadata.

### CODE REVIEW

File: app/services/BackendChecksService.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    val url = s"${configuration.get[String]("backendchecks.baseUrl")}/backend-checks/$consignmentId"
+    wsClient
+      .url(url)
+      .addHttpHeaders(("Authorization", token), ("Content-Type", "application/json"))
+      .post("{}")
+      .flatMap(r =>
+        r.status match {
+          case 200 => Future(true)
+          case _ =>
+            logger.error(s"Export api response ${r.status} ${r.body}")
```
Comment (TomJKing) [inline app/services/BackendChecksService.scala:22]: Is that the correct log error, as this is about the backend checks, not the export api?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor comment.

File: app/services/BackendChecksService.scala (hunk 1 above)
Comment (MancunianSam) [reply app/services/BackendChecksService.scala:22]: It isn't, I copied it from the export class. I've changed it.

**Comment by MancunianSam** [comment]
There has been some javascript changes since you last looked at this if you want to check those over as well.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**